cleansweep/plugins/committees/forms.py

```diff
@@ -53,6 +51,29 @@ def ensure_empty_slots(self, n=5):
         for i in range(n-empty_slots):
             self.roles.append_entry()
 
+
+class NewCommitteeForm(CommitteeForm):
+    """
+    Creates a form for adding a new committee structure.
+    """
+    level = StringField('Level')  # Level field at which the committee structure is getting created.
+
+    def __init__(self, place, level, *a, **kw):
+        super(NewCommitteeForm, self).__init__(place, *a, **kw)
+        self.level.data = level
+
+
+class EditCommitteeForm(CommitteeForm):
+    """
+    Edit form for editing committee structures. Extends CommitteeForm.
+    """
+    level = SelectField('Level', choices=[])  # Select field for place level selection.
+
+    def __init__(self, place, *a, **kw):
+        super(EditCommitteeForm, self).__init__(place, *a, **kw)
+        place_types = [place.type] + place.type.get_subtypes()
+        self.level.choices = [(t.short_name, t.name) for t in place_types]
+
```

Comment:
Do we really need two different forms here (three if you count the base class)? Why not get rid of the level dropdown completely. For new committees, the level is already available from the URL and for the existing committees, we don't allow changing that.